android/app/src/profile/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.example.mobile">
+    package="de.dealog.mobile.pilot">
```

Comment:
Do we want to go with the `pilot` suffix?

Reply:
Just added it to have a package for the pilot version, which we are working on for now. This allows us to have two versions in the play store, a released version for the "pilot" and a closed beta supporting the whole country.

Reply:
Other possibility is to remove `pilot` and use a beta channel for the full release later. When we make that available, the pilot users will be updated to the full version.

Reply:
Leave it for now. We can change this also later.